code/modules/mob/mob.dm

```diff
@@ -317,6 +317,27 @@
 	return
 */
 
+/mob/living/verb/set_clumsiness()
+	set name = "Set Clumsiness"
+	set category = "IC"
+	set desc = "Toggles whether or not your character is clumsy and will fail at pouring stuff without warning."
+	var/list/modes = list("Holders", "Holders/Placeables", "Clumsy")
+	var/selection = input("Set clumsiness:", "Sensitivity", modes[src.clumsy+1]) in modes
+
+	if (selection == modes[1])
+		usr<<"You will no longer miss loadables and fillables without confirmation."

```

Comment:
No need to use ``usr`` here, the message can be sent to ``src``, which is more safe. It probably doesn't matter here but this would runtime if something that lacks ``usr`` called this verb.